build-tools/packages/build-infrastructure/src/buildProject.ts

```diff
@@ -237,56 +374,3 @@ export function getAllDependencies(
 		workspaces: [...workspaces],
 	};
 }
-
-/**
- * Sets the dependency range for a group of packages given a group of dependencies to update.
- * The changes are written to package.json. After the update, the packages are
- * reloaded so the in-memory data reflects the version changes.
- *
- * @param packagesToUpdate - A list of objects whose version should be updated.
- * @param dependencies - A list of objects that the packagesToUpdate depend on that should have updated ranges.
- * @param dependencyRange - The new version range to set for the packageToUpdate dependencies.
- */
-export async function setDependencyRange<P extends IPackage>(
```

Comment:
This function isn't used and is the only function that depends on the version-tools package. Now that build-cli imports from build-infra, though, I can move this function to build-cli.